content/en/mobile_testing/settings/_index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Mobile Application Testing Settings
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/mobile_testing/mobile_app_tests"
+  tag: "Documentation"
+  text: "Learn how to create a mobile test"
+- link: "/continuous_testing/cicd_integrations"
+  tag: "Documentation"
+  text: "Run your Synthetic tests in a CI pipeline"
+---
+{{< jqmath-vanilla >}}
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeHny7qHl5w3u3DCI4Ilc-r4IQZSAFOeZgMvP3CKBO9hEl1qA/viewform" >}}
+Mobile Application Testing is in private beta. Register to request access!
+{{< /callout >}} 
+
+## Overview
+
+Manage your uploaded mobile applications and your parallelization settings on the [Synthetic Monitoring & Continuous Testing Settings page][1].
+
+{{< img src="mobile_testing/applications_list.png" alt="Mobile Applications Settings" style="width:100%;">}}
+
+## Create an application
+
+To add a mobile application, navigate to the [**Mobile Applications List** tab][5] and click **+ Create Application**.
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+1. Select **Android** as the OS for your mobile application.
+2. Name your mobile application.
+3. Add `env` tags as well as additional tags to your mobile application. You can use these tags to quickly filter through your mobile app tests on the [Synthetic Monitoring & Continuous Testing page][101]. 
+4. Optionally, enter a description for your mobile application.
+5. Upload an [`.apk` file][102].
+6. Enter a name for the version of your mobile application. Optionally, select **Mark this version as latest**.
+7. Click **Create Application**.
+
+[101]: https://app.datadoghq.com/synthetics/tests
+[102]: https://developer.android.com/tools/bundletool
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+1. Select **iOS** as the OS for your mobile application.
+2. Name your mobile application.
+3. Add `env` tags as well as additional tags to your mobile application. You can use these tags to quickly filter through your mobile app tests on the [Synthetic Monitoring & Continuous Testing page][101]. 
+4. Optionally, enter a description for your mobile application.
+5. Upload an `.ipa` file.
+6. Enter a name for the version of your mobile application. Optionally, select **Mark this version as latest**.
+7. Click **Create Application**.
+
+[101]: https://app.datadoghq.com/synthetics/tests
+
+{{% /tab %}}
+{{< /tabs >}}
+
+To edit or delete a mobile application, hover over a mobile application in the **Mobile Applications List** and click on the respective icon.
+
+## Manage application versions
+
+Clicking on a mobile application in the **Mobile Applications List** displays existing versions of the application. Hover over a version and click the **+** icon to [create a mobile app test][6] with the selected mobile application's version.
+
+To edit or delete a version of a mobile application, hover over a version in the mobile application and click on the respective icon.
+
+## Add a version
```

Comment:
Should be under Manage application versions header

Reply:
```suggestion
### Add a version
```